librz/core/cmd/cmd_interpret.c

```diff
@@ -68,3 +68,8 @@ RZ_IPI RzCmdStatus rz_interpret_macro_handler(RzCore *core, int argc, const char
 	rz_cmd_macro_call(&core->rcmd->macro, argv[1]);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_interpret_macro_multiple_handler(RzCore *core, int argc, const char **argv) {
+	rz_cmd_macro_call_multiple(&core->rcmd->macro, argv[1]);
```

Comment:
how does `rz_cmd_macro_call_multiple` take the other arguments passed to the command?

Reply:
I see unfortunately `rz_interpret_macro_handler` does not pass the rest of `argv` either... but probably the API of rz_macro should be rewritten completely to avoid all the input parsing it does :(

Reply:
I agree that the parsing should be improved but I probably won't have the time to do that